src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/clients/CcdEventConsumer.java

```diff
@@ -74,44 +69,24 @@ protected void consumeMessage(ServiceBusSessionReceiverClient sessionReceiver) {
                             log.info(String.format("Processing completed successfully"
                                                    + " on case details: %s", loggerMsg));
                         } catch (JsonProcessingException exp) {
-                            handleJsonError(receiver, message, loggerMsg, incomingMessage, exp);
-                        } catch (ResourceAccessException exp) {
-                            handleApplicationError(receiver, message, loggerMsg, incomingMessage, exp);
+                            ccdEventErrorHandler.handleJsonError(
+                                receiver, message, loggerMsg, incomingMessage, exp
+                            );
+                        } catch (RestClientException exp) {
+                            ccdEventErrorHandler.handleApplicationError(
+                                receiver, message, loggerMsg, incomingMessage, exp
+                            );
+                        } catch (Throwable exp) {
```

Comment:
Probably better to catch `Exception` rather than `Throwable` since it includes `Error` which you don't really want to handle.